src/TelemetryChannels/ServerTelemetryChannel/NuGet/TelemetryChannel.NuGet.csproj

```diff
@@ -38,7 +38,7 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Core\Managed\Net45\Core.Net45.csproj">
+    <ProjectReference Include="..\..\..\Core\Managed\Net40\Core.Net40.csproj">
```

Comment:
Why are these changing from Net45 to Net40 when the target framework version is still 4.5?
"<TargetFrameworkVersion>v4.5</TargetFrameworkVersion>"